src/Cylinder.cc

```diff
@@ -15,6 +15,7 @@
  *
 */
 #include <sstream>
```

Comment:
```suggestion
#include <optional>
#include <sstream>
```

Reply:
Added in [1f16e1f](https://github.com/gazebosim/sdformat/pull/1304/commits/1f16e1f0a5454acc5a30a4b86b8f273b1535eb95)